supervisor/plugins/dns.py

```diff
@@ -280,6 +282,11 @@ def _write_config(self) -> None:
         if not self._loop:
             dns_servers = self.servers
             dns_locals = self.locals
+            # Allow a DNS server with an issue, just tell user about it
+            await asyncio.gather(
+                *[self._check_server(server) for server in self.servers + self.locals],
+                return_exceptions=True,
```

Comment:
Let's run that on the resolution center check. That allows also to remove the issue once the server is online again or fixed

Reply:
@pvizeli gave each of the issues their own check, let me know if this works.

Reply:
looks great, thanks